File: Bars.Gkh/Entities/Dicts/WorkKindCurrentRepair.cs
namespace Bars.Gkh.Entities
{
    using Bars.Gkh.Enums;

    public class WorkKindCurrentRepair : BaseGkhEntity
    {
        /// <summary>
        /// Наименование
        /// </summary>
        public virtual string Name { get; set; }

        /// <summary>
        /// Код
        /// </summary>
        public virtual string Code { get; set; }

        /// <summary>
        /// Единица измерения
        /// </summary>
        public virtual UnitMeasure UnitMeasure { get; set; }

        /// <summary>
        /// Тип работы
        /// </summary>
        public virtual TypeWork TypeWork { get; set; }
    }
}

File: Bars.Gkh/libs/B4/model/Dict/WorkKindCurrentRepair.js
Ext.define('B4.model.dict.WorkKindCurrentRepair', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'WorkKindCurrentRepair'
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'Name' },
        { name: 'Code' },
        { name: 'UnitMeasure', defaultValue: null },
        { name: 'TypeWork', defaultValue: 10 }
    ]
});